recipe/meta.yaml

```diff
@@ -39,7 +35,9 @@ requirements:
     - libblas
     - liblapack
     - metis
+    - ampl-mp
   run:
+    - ampl-mp
```

Comment:
Checking to see if this needs to be in run. Probably not.

Reply:
nope: https://github.com/conda-forge/ampl-mp-feedstock/blob/master/recipe/meta.yaml#L18